project.yml

```diff
@@ -76,8 +76,8 @@ targets:
         com.apple.security.network.server: true
         com.apple.developer.exposure-notification: true
     configFiles:
-      Debug: Configs/Prod-signing.xcconfig
-      Release: Configs/Prod-signing.xcconfig
+      Debug: Configs/Templates/Prod-signing.xcconfig
```

Comment:
Just fix the paths.

Reply:
Point of it it's to copy template and create your own. Becouse you need to set your own provision profiles. So you will be able to run it on device.

Reply:
from readme:

**Code signing**
To update your code signing settings, you would need to copy template .xcconfig files from Configs/Templates/ directory to Configs/ directory and fill them with proper values. xcodegen is looking for these files in Configs directory and use them in project generation. All .xcconfig files in Configs directory are ignored by git, so you don't have to worry about accidentally pushing your code signing settings or pulling somebody elses.

Reply:
Hello @augard 
Sorry for the delay. My suggestion is to define configs path as a constant, for example using `export EROUSKA_CONFIGS_PATH="My/Path/For/Configs"` (maybe with the default value "Configs/Templates"). If you agree with that, I can also fix README.md. If not, I will just revert the changes I did in project.yml.